include/library/spdm_common_lib.h

```diff
@@ -114,6 +114,10 @@ typedef enum {
 
     LIBSPDM_DATA_APP_CONTEXT_DATA,
 
+    /*The BIT0 control to generate SPDM_ERROR_CODE_DECRYPT_ERROR response or drop the request silently.*/
+    /*If the BIT0 is not set, generate SPDM_ERROR_CODE_DECRYPT_ERROR response.*/
+    LIBSPDM_DATA_HANDLE_ERROR_RETURN_POLICY,
```

Comment:
Please state what if BIT0 is set, what happens.

Reply:
OK. I have added the comments.